src/ui/FieldTable/FieldValue.ts

```diff
@@ -126,7 +127,12 @@ export class FieldValue extends Component {
         base: ComponentOptions.buildNumberOption(showOnlyWithHelper(['size'], { min: 0, defaultValue: 0 })),
         isMilliseconds: ComponentOptions.buildBooleanOption(showOnlyWithHelper(['timeSpan'])),
       }
-    })
+    }),
+
+    /**
+     * Specify a caption to display before the value. <br/>
+     */
+    textCaption: ComponentOptions.buildStringOption()
```

Comment:
Should it be a localized string ? I'm guessing it should be. In that case, use `ComponentOptions.buildLocalizedStringOption()`